File: src/main/java/me/waiaf/skytizencore/commands/CreativeCommand.java
package me.waiaf.skytizencore.commands;

import org.bukkit.ChatColor;
import org.bukkit.GameMode;
import org.bukkit.entity.Player;

@CommandInfo(name = "creative", permission = "rank.builder", requiresPlayer = true)
public class CreativeCommand extends PluginCommand {

    @Override
    public void execute(Player player, String[] args) {

        player.setGameMode(GameMode.CREATIVE);
        player.sendMessage(ChatColor.GREEN + "Đã chuyển sang chế độ sáng tạo");

    }

}

File: src/main/java/me/waiaf/skytizencore/commands/CreativeCommand.kt
package me.waiaf.skytizencore.commands

import org.bukkit.ChatColor
import org.bukkit.GameMode
import org.bukkit.entity.Player

@CommandInfo(name = "creative", permission = "rank.builder", requiresPlayer = true)
class CreativeCommand : PluginCommand() {
    override fun execute(player: Player, args: Array<String>?) {
        player.gameMode = GameMode.CREATIVE
        player.sendMessage(ChatColor.GREEN.toString() + "Đã chuyển sang chế độ sáng tạo")
    }
}